fixture_magic/management/commands/custom_dump.py

```diff
@@ -14,16 +12,35 @@
     from django.apps import apps as loading
 from django.core.serializers import serialize
 from django.conf import settings
-from django.template import Variable, VariableDoesNotExist
 
 from fixture_magic.utils import (
     add_to_serialize_list,
     reorder_json,
-    serialize_me,
     serialize_fully
 )
 
 
+def process_dep(parent, dep, serialize_me, seen):
+    parts = dep.split('.')
+    current = parts.pop(0)
+    remain = '.'.join(parts)
+
+    try:
+        thing = getattr(parent, current)
+    except AttributeError:
+        pass  # related object not found
+    else:
+        if hasattr(thing, 'all'):
+            children = thing.all()
+        else:
+            children = [thing]
+        add_to_serialize_list(children, serialize_me, seen)
+
+        if remain:
```

Comment:
remain?

Also, this being true depends on `len(parts) > 0`?

Reply:
Will rename.

Yep, it does.